agent/wpthook/test_state.h

```diff
@@ -35,7 +35,8 @@ class DevTools;
 class Trace;
 
 const int TEST_RESULT_NO_ERROR = 0;
-const int TEST_RESULT_TIMEOUT = 99997;
+const int TEST_RESULT_OVERALL_TIMEOUT_REACHED_ERROR = 99995;

```

Comment:
99996 was previously defined but then removed. There are traces of it in comments throughout the code. I made it 99995 so that I don't interfere with that previous value.

See: https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/result-codes